### ISSUE TITLE
Add something like javascript

### ISSUE THREAD

**Issue by Luca0208** [BTCTaras/kristscape#7]
Sandboxed lua with a powerful api might be a good idea, if you want I can give it a try and PR.

**Comment by Luca0208** [comment]
Is someone even reading this?

**Comment by Lustyn** [comment]
Yes. But Taras is working on other stuff right now and kristscape isn't in
a state that would allow this to be added.

**Comment by bauen1** [comment]
How about people trying to make a sandbox and other people try to unsandbox it, doing this a few times would make a good sandbox wich could be included in kristscape.
Because making a good sandbox is a difficulty task.

**Comment by Luca0208** [comment]
I asked the reading thing because no one bothered closing this either.

**Comment by Luca0208** [comment]
Also I am working on koogle. You might've been able to guess from the name what it does :P 
So that's why this would be useful.

**Comment by bauen1** [comment]
Hm, let me thing about what google could be doing.
(Dammit spell correction revealed the secret)

**Comment by Luca0208** [comment]
I'll start writing some code for KSML and PR

**Comment by Luca0208** [comment]
Ok, I give that thing with KSML up the current code works like 0% and I won't start from 0. I guess this issue can be closed?

**Comment by BTCTaras** [comment]
No we can reopen this, this type of functionality may come sooner than expected

**Comment by Luca0208** [comment]
Ok.

**Comment by bauen1** [comment]
Just wanna add this here for the sandbox, whitelist rather than blacklist and maybe don't even add stuff like ragweed etc. because it doesn't matter if the sandbox is detected.
I think this should go in its own file like `ksml.lua`, what would be an appropiate name?
My suggestion would be `kssl` but i think that seems to go in the https direction

**Comment by Luca0208** [comment]
well my idea would be something like a [lua] tab and then you could use some OOP style stuff

**Comment by Luca0208** [comment]
Wrong button

**Comment by BTCTaras** [comment]
It would most likely end up being `[SCRIPT:LUA]`, so that the argument passed to the `SCRIPT` tag is the script's language

**Comment by Luca0208** [comment]
Well what other languages are there?

**Comment by BTCTaras** [comment]
`INQUIRE`, which I've designed explicitly to function alongside KSML a few months ago. I'll be rewriting it before it gets in the repo though.

**Comment by Luca0208** [comment]
Nice more than 1 language to choose from. I can't wait for kristscape to come out, so I can finish koogle. Btw if anyone is interested in koogle write me a PM.
